build.Dockerfile

```diff
@@ -18,8 +18,10 @@
 
 ### Cached Builder Stage
 ##################################################
-# For the OLI guys, we removed the sccache part is it makes the native GA runners run out of space.
```

Comment:
Please leave the comment I left for you guys. 😄 hence, you can remove the sccache installation from below.

Reply:
I'll keep the comment.